kedro/contrib/io/azure/csv_blob.py

```diff
@@ -107,30 +111,60 @@ def __init__(
                 Here you can find all available arguments:
                 https://pandas.pydata.org/pandas-docs/stable/generated/pandas.DataFrame.to_csv.html
                 All defaults are preserved, but "index", which is set to False.
+            version: If specified, should be an instance of
+                ``kedro.io.core.Version``. If its ``load`` attribute is
+                None, the latest version will be loaded. If its ``save``
+                attribute is None, save version will be autogenerated.
 
         """
-        self._filepath = filepath
         self._container_name = container_name
         self._credentials = credentials if credentials else {}
         self._blob_to_text_args = blob_to_text_args if blob_to_text_args else {}
         self._blob_from_text_args = blob_from_text_args if blob_from_text_args else {}
-        super().__init__(load_args, save_args)
+        self._blob_service = BlockBlobService(**self._credentials)
+        super().__init__(
+            PurePath(filepath),
+            version,
+            exists_function=self._exists_blob,
```

Comment:
`self._exists_blob` method expects `blob_name` as an argument, but you don't pass it here.

Reply:
D'you mind clarifying this a tiny bit? I'm probably just being silly, but how do I pass the argument? It expects a callable?